src/EFCore.PG/Extensions/NpgsqlIndexBuilderExtensions.cs

```diff
@@ -47,5 +47,24 @@ public static IndexBuilder ForNpgsqlHasOperators([NotNull] this IndexBuilder ind
 
             return indexBuilder;
         }
+
+        /// <summary>
+        /// The PostgreSQL index sort options to be used.
+        /// </summary>
+        /// <remarks>
+        /// https://www.postgresql.org/docs/current/static/indexes-ordering.html
+        /// </remarks>
+        /// <param name="indexBuilder"> The builder for the index being configured. </param>
+        /// <param name="sortOptions"> The sort options to use for each column. </param>
+        /// <returns> A builder to further configure the index. </returns>
+        public static IndexBuilder ForNpgsqlHasSortOptions([NotNull] this IndexBuilder indexBuilder, [CanBeNull] params string[] sortOptions)
```

Comment:
Not sure which it is, but could you add `[ItemCanBeNull]` or `[ItemNotNull]` to `sortOptions`?